boa3/internal/cli_commands/compile_command.py

```diff
@@ -37,6 +37,9 @@ def add_arguments_and_callback(self):
         self.parser.add_argument("--no-failfast",
                                  action='store_true',
                                  help="Do not stop on first compile error")
+        self.parser.add_argument("--log-level",
+                                 type=str,
+                                 help="Log output level")
```

Comment:
The user won't know which types of log level they can use. You could add a `choice`, like:
```python
        self.parser.add_argument("--log-level",
                                 type=str,
                                 choices=['INFO', 'WARNING', 'ERROR'],
                                 help="Log output level")
```

or put it on the 'help' message

Reply:
I don't think we need to include choices since this `--log-level` flag is pretty common on Python programs. 
Also `INFO`, `WARNING` and `ERROR` are not the only log level options and the ``logging` package allows to create log levels different from the default ones, so I don't think it's a good idea to limit the choices on cli.

I will include a handle for invalid log levels passed to the compiler